examples/columns2.py

```diff
@@ -0,0 +1,56 @@
+"""
+Builds calendar layout using Columns and Tables.
+Usage:
+python columns2.py [YEAR]
+Example:
+python columns2.py 2021
+"""
+import argparse
+
+import calendar
+from datetime import datetime
+
+from rich.table import Table
+from rich.columns import Columns
+from rich import print
+
+
+def display_calendar(year):
+
+    today = datetime.today()
+    year = int(year)
+    cal = calendar.Calendar()
+
+    tables = []
+    for month in range(1, 13):
+
+        table = Table(title=f"""{calendar.month_name[month]} {year}""", style="green")
+
+        for wd in cal.iterweekdays():
```

Comment:
Can we call this `week_day`. I have a no abbreviation policy.

Reply:
done